<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="main_layout">
   <property name="margin">
    <number>5</number>
   </property>
   <item row="2" column="0">
    <widget class="QGroupBox" name="config_group">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Configure Equipment</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="config_stage">
        <property name="text">
         <string>Config.
Stage</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="config_light_source">
        <property name="text">
         <string>Config.
Light Source</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="config_spectrometers">
        <property name="text">
         <string>Config.
Spectrometers</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="open_browser">
        <property name="text">
         <string>Open Data
Browser</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Hyperspectral Controls</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="scan_description_label">
        <property name="text">
         <string>Scan Description:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="scan_description"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="stage_select_layout">
        <item>
         <widget class="QLabel" name="stage_select_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Stage Select:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="stage_select"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="wavelength_layout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="wavelength_label">
          <property name="text">
           <string>Wavelength:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="wavelength_range">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="view_wavelength">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layer_layout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="layer_label">
          <property name="text">
           <string>Layer:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="view_layer"/>
        </item>
        <item>
         <widget class="QCheckBox" name="override_view_layer">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="safe_exit">
        <property name="text">
         <string>Safe Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="gridscanner_widget" native="true"/>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QWidget" name="figure_widget" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>300</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
